Makefile

```diff
@@ -1,36 +1,48 @@
+SOURCE_FILES?=./...
 TEST_PATTERN?=.
 TEST_OPTIONS?=
 
+export PATH := ./bin:$(PATH)
+export GO111MODULE := on
+export GOPROXY = https://proxy.golang.org,direct
+
+GOBIN?=$(shell go env GOBIN)
+GOLANGCI_LINT := $(GOBIN)/golangci-lint
+$(GOLANGCI_LINT): 
+	cd "$$(mktemp -d)" && \
+		go get github.com/golangci/golangci-lint
```

Comment:
maybe use the `tools.go` approach? e.g. https://github.com/goreleaser/goreleaser/blob/master/tools.go

Reply:
@caarlos0 happy to switch to that if you'd prefer?

Reply:
if your way its working, its fine by me too :)

Reply:
ok cool, will merge